src/ec/packages/ec-component-accordion/ecl-accordion.html.twig

```diff
@@ -66,6 +66,7 @@
             { name: 'data-ecl-accordion-toggle' },
           ]
         } only %}
+        
```

Comment:
but i remember you were adding the "-content" to the ids like in ECL in the accordion, why did we remove that? (mine was just a comment, it is a bit weird but as i wrote maybe there was a reason behind that.